apps/sms/views/conversation/js/information.js

```diff
@@ -247,14 +248,17 @@ var VIEWS = {
 
         ConversationView.promptContact({
           number: target.dataset.number
-        });
+        }).then(
+          () => Navigation.toPanel('thread', { id: this.id })
+        );
```

Comment:
Ah, nice, thanks!

__unrelated note:__ IMO this behavior with automatically returning to Conversation view and that it's called "multiple recipient overlay" on page 13 makes me think about Group view like about Group dialog that is automatically dismissed on action :)